spec/unit/pops/validator/validator_spec.rb

```diff
@@ -128,19 +86,25 @@ def validate(model)
       expect(acceptor.error_count).to eql(0)
       expect(acceptor).to_not have_issue(Puppet::Pops::Issues::DUPLICATE_KEY)
     end
+  end
 
-    it 'does not produce an error for duplicate default in a case expression' do
-      acceptor = validate(parse('case 1 { default: {1} default : {2} }'))
-      expect(acceptor.warning_count).to eql(0)
-      expect(acceptor.error_count).to eql(0)
-      expect(acceptor).to_not have_issue(Puppet::Pops::Issues::DUPLICATE_DEFAULT)
-    end
+  [:off, :warning, :error].each do |level|
```

Comment:
Why test this with different strict levels if the error isn't controlled by strict?